openapi.yaml

```diff
@@ -10692,6 +10692,22 @@ paths:
                   type: string
                   description: The label for this keypair, for display purposes only.
                   example: my-key
+                bucket_access:
+                  type: object
+                  description: Defines this key as a limited access key, and contains values pertaining to permissions. Limited Access Keys block access to all buckets except for those specified within the object.
+                  properties:
+                    cluster:
+                      type: string
+                      description: The datacenter where a bucket with limited access is hosted.
```

Comment:
It would be more accurate to say `The Object Storage cluster where a bucket to which the key should have access is hosted.`

The OBJ cluster label is different than the datacenter/region label (typically just a `-1` appended to the datacenter name.)